docker/ccf_ci_built

```diff
@@ -2,39 +2,13 @@
 # Contains CCF build dependencies and toolchain for target platform
 # Also contains CCF source and build directory
 
+# Latest image as of this change
 ARG platform=sgx
+ARG base=ccfmsrc.azurecr.io/ccf/ci:oe-0.18.4-dcap-ql-dev
```

Comment:
Can we pass this in as a build arg? That would make sure it stays in sync with the CI file above.

Reply:
Sure
https://github.com/microsoft/CCF/pull/4737/commits/8be36f559d80e9961437f099ae9f1bb6e54d09c2